numpy/core/arrayprint.py

```diff
@@ -677,8 +677,15 @@ def recurser(index, hanging_indent):
                 word = recurser(index + (-i,), next_hanging_indent) + separator
                 s, line = _extendLine(s, line, word, max_line_len, hanging_indent)
```

Comment:
This line and a few others from the last PR are longer than 80 chars. Rename `max_line_len`to something shorter?